book/02-using-atom/sections/06-customizing.asc

```diff
@@ -217,3 +219,20 @@ The scope name is shown in the settings view for each language. Search for the l
 
 .Finding a language grammar
 image::images/python-grammar.png[python grammar]
+
+[[_customizing_language_recognition]]
+==== Customizing Language Recognition
+
+If you want to customize which language package is used when loading files with a certain extension, you need only manually edit your Atom `config.cson` file. You can open it using the _Application: Open Your Config_ command from the Command Palette. For example, if you wanted to add the `foo` extension to the CoffeeScript language, you could add this to your configuration file under the `*.core` section:
+
+[source,javascript]
+----
+'*'
+  core:
+    customFileTypes:
+      'source.coffee': [
+        'foo'
+      ]
+----
+
+In the example above, `source.coffee` is the language's scope name (see <<_finding_the_language_scope_name>> for more information) and `foo` is the file extension to match without the period. Adding a period to the beginning of either of these will not work.
```

Comment:
Thank you, this explicitness is great for quickly understanding... I guess it's confusing because many of the scopes are the same as the file extensions, so that's a good example.